randovania/games/samus_returns/logic_database/Area 2 - Dam Exterior.txt

```diff
@@ -563,24 +587,20 @@ Extra - asset_id: collision_camera_028
   * Extra - start_point_actor_name: ST_Alpha_003_Checkpoint
   > Door to Metroid Caverns Lobby
       Any of the following:
-          High Jump Boots or Gravity Suit or Space Jump or Use Spider Ball
+          High Jump Boots or Gravity Suit or Space Jump or Single-wall Wall Jump (Beginner) or Use Spider Ball
```

Comment:
what about super jumps / morph extends?

Reply:
I did manage to get an Unmorph Extend once from outside the water, but I wasn't recording. It's so stupidly precise. As for a combination of Super Jump/Unmorph Extend, doesn't seem possible. Super Jump on its own doesn't work either.